src/frontend/src/catalog/system_catalog/rw_catalog/rw_description.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::iter;
+
+use risingwave_common::catalog::RW_CATALOG_SCHEMA_NAME;
+use risingwave_common::error::Result;
+use risingwave_common::row::OwnedRow;
+use risingwave_common::types::{DataType, ScalarImpl};
+
+use crate::catalog::system_catalog::{BuiltinTable, SysCatalogReaderImpl};
+
+pub const RW_DESCRIPTION: BuiltinTable = BuiltinTable {
+    name: "rw_description",
```

Comment:
Could you please change `pg_description` as a view upon this system table?

Reply:
Do you think the current `pg_description` implementation is reasonable... I may need to update e2e_test to synchronize the changes in `pg_description`.

Reply:
The current `pg_description` implementation is just a kind of workaround. Plz also update related e2e tests as well.